### ISSUE TITLE
VSP Accessibility feedback on GIBCT Estimate Your Benefits [Usability Testing Prep]

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8819]
* I think the estimated benefits drawer should be a `<button> + <div>` that shows/hides a regular accordion. It should have a few key ARIA attributes:
  * `aria-expanded="false || true"`
  * `aria-controls="UNIQUE_ID"` The unique ID should be associated with the div of benefits. 
  * `aria-controls` should only appear when the drawer is open, otherwise it should be a **null** value or we'll get an axe accessibility error.
* I'm making an educated guess that the drawer will slide into place as users scroll to the Estimate Your Benefits section. It might be good to consider a `prefers-reduced-motion` query to reduce or eliminate the animation, and have the drawer snap into place rather than slide in. MDN has a good introductory article on [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) and support for the spec.
* I have a few ideas about where we should place the drawer in the source order, but can't give my best guidance until I'm able to interact with it on Staging. I'll hold this guidance until our Staging Review, or until the functionality is built out and somewhat code stable.
* If it's practical to do so, the full estimated benefits panel should always render on mobile devices, and be available for users to scroll to. By having that as a default, we have a graceful fallback for older or low-powered devices running browsers like Opera Mini.

**Comment by dneel-bah** [comment]
Items 1 & 2 are to be covered in #9191.
We will wait for your feedback on # 3 in the staging review.
Item # 4 is current behavior.

**Comment by desireeturner** [comment]
@1Copenut The outstanding issue keeping this open has been tested and closed (#9191 ). Please take a look and let us know if anything else is needed before closing. Thanks!

**Comment by dneel-bah** [comment]
@1Copenut 
All items are available on staging.

Note; the sheet behavior is under consideration to be tweaked about when it gets displayed, but you can review it for source order recommendations.  @amyknox can provide additional details about the forthcoming tweaks in the near future.

**Comment by 1Copenut** [comment]
@dneel-bah the only thing I can see is there's a blue bar appearing to the left of some of the expanded section inputs. Screenshot attached below, and adding @emilywaggoner && @CrystabelReiter for design awareness.

---
![Screen Shot 2020-06-03 at 11 30 33 AM](https://user-images.githubusercontent.com/934879/83663160-15ecc900-a58e-11ea-8f93-f33825a7fd05.png)

**Comment by CrystabelReiter** [comment]
I've only seen the bar for additional info components or conditional questions, when the answer to the first question determines the next question to be asked. If that doesn't apply here, I'd propose removing it. @emilywaggoner  @1Copenut

**Comment by dneel-bah** [comment]
@CrystabelReiter The questions following "Which GI bill do you want to use" change conditionally based on the answer to that question.  That answer is pre-selected based on inputs on previous pages hence the blue bar display by default.  Is that an issue that needs to change?

**Comment by CrystabelReiter** [comment]
@dneel-bah I see what you mean, I just went through the form itself. I think the blue bar seems appropriate here. Thanks for clarifying.

**Comment by dneel-bah** [comment]
@1Copenut I think this got lost in the shuffle.  

Confirmed with Crystabel that the blue bar is good. 

The issue keeping this open has been tested and closed (#9444). Please take a look and let us know if anything else is needed before closing. Thanks!

**Comment by 1Copenut** [comment]
@dneel-bah Yes, let's go ahead and close this item per our discussion last week about the mobile-only tray being a separate feature release.

**Comment by dneel-bah** [comment]
Closing this issue as recommended by previous comment.
